libr/anal/meta.c

```diff
@@ -52,13 +54,15 @@ static char *meta_inrange_get (RAnal *a, ut64 addr, int size) {
 }
 
 static bool meta_inrange_add (RAnal *a, ut64 addr, int size) {
+	if (size <= 0) {
```

Comment:
just < 1

Reply:
0 is more natural than 1. `<= 0` is also more akin to what optimized codegen does.